wren-ui/migrations/20240928165009_add_model_nested_column.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('model_nested_column', (table) => {
+    table.increments('id').comment('ID');
+    table.integer('model_id').comment('Reference to model ID');
+    table.integer('column_id').comment('Reference to column ID');
```

Comment:
The column_id was used in the service layer. 
If you would like to aggregate the column & nested column with the `column_path` and `model_id` (eg: mdlBuilder.ts file), then I think the column_id can be removed.
On the other hand, you can keep the column_id and use this column to aggregate. 
<img width="863" alt="image" src="https://github.com/user-attachments/assets/c6e75411-3096-40dc-996f-3f2cc545c9b8">